### PR TITLE
6 patch

### PR DESCRIPTION
**Pull request by olivershh**
Is the test on line 99 OK? I could add a separate db query to check that it has updated but not sure if it's needed as the return comes from the query.

### CODE REVIEW

**Comment by olivershh** [comment]
I could also test for negative numbers but might be overkill?

File: __tests__/app.test.js [hunk 1]
```diff
@@ -83,15 +83,14 @@ describe("/api/articles/:article_id", () => {
...
       return request(app)
         .patch("/api/articles/2")
         .send({ inc_votes: 10 })
         .expect(201)
         .then((response) => {
           const { body } = response;
-          console.log(body);
           expect(body).toEqual(
             expect.objectContaining({ article: expect.any(Object) })
           );
```
Comment (popatre) [inline __tests__/app.test.js:96]: Dont really need this one as the one below covers it in more detail

File: __tests__/app.test.js [hunk 2]
```diff
@@ -123,7 +122,26 @@ describe("/api/articles/:article_id", () => {
...
+      return request(app)
+        .patch("/api/articles/10;")
+        .send({ inc_votes: "notanumber" })
+        .expect(400)
+        .then((response) => {
+          const { body } = response;
+          expect(body).toEqual({ msg: "bad request" });
+        });
+    });
   });
```
Comment (popatre) [inline __tests__/app.test.js:145]: Could do with a another test for when the object is missing the inc_votes key

File: controllers/articles.controller.js [hunk 3]
```diff
@@ -17,7 +17,9 @@ exports.patchArticleById = (req, res, next) => {
   const articleID = req.params.article_id;
   const votes = req.body.inc_votes;
 
-  updateArticleById(articleID, votes).then((article) => {
-    res.status(201).send({ article: article });
-  });
+  updateArticleById(articleID, votes)
+    .then((article) => {
+      res.status(201).send({ article: article });
```
Comment (popatre) [inline controllers/articles.controller.js:22]: should be 200 - as a resource has not been created, just updated

File: __tests__/app.test.js [hunk 4]
```diff
@@ -83,6 +83,48 @@ describe("/api/articles/:article_id", () => {
...
+        .send({ inc_votes: 10 })
+        .expect(201)
+        .then((response) => {
+          const { body } = response;
+          console.log(body);
+          expect(body).toEqual(
+            expect.objectContaining({ article: expect.any(Object) })
+          );
+        });
+    });
```
Comment (popatre) [inline __tests__/app.test.js:98]: Not needed

File: __tests__/app.test.js [hunk 5]
```diff
@@ -83,6 +83,48 @@ describe("/api/articles/:article_id", () => {
...
+          const { body } = response;
+          expect(body.article).toHaveProperty("author", expect.any(String));
+          expect(body.article).toHaveProperty("title", expect.any(String));
+          expect(body.article).toHaveProperty("article_id", 4);
+          expect(body.article).toHaveProperty("body", expect.any(String));
+          expect(body.article).toHaveProperty("topic", expect.any(String));
+          expect(body.article).toHaveProperty("created_at", expect.any(String));
+          expect(body.article).toHaveProperty("votes", 100);
+        });
+    });
```
Comment (popatre) [inline __tests__/app.test.js:114]: This does the same as the above test, but better

**Review by popatre — CHANGES_REQUESTED** [review]
just needs another test and that other test isnt needed

File: controllers/articles.controller.js (hunk 3 above)
Comment (olivershh) [reply controllers/articles.controller.js:22]: Done.

File: __tests__/app.test.js (hunk 1 above)
Comment (olivershh) [reply __tests__/app.test.js:96]: Deleted. (not sure if you still get these notifications? I'm just using this as a ticklist otherwise)
